Automattic/wp-calypso
# Emails: Initial code to create new comparison component

#### Changes proposed in this Pull Request

Initial code for a new comparison component in Emails Comparison page.

#### Testing instructions

1. Go to a site with domains without email subscription
2. Click on 'Add email' for the desired site
3. Once you are in the comparison page, add at the end the following flag: `?flags=emails/new-email-comparison`
4. Click then in the "see how they compare" in the top of the screen

![image](https://user-images.githubusercontent.com/5689927/147093087-486da1fe-4ce3-4391-be19-698d305647e6.png)

Related to {1200182182542585-as-1201334995813848}
